iPetCare/API/Controllers/PetsController.cs

```diff
@@ -19,13 +21,33 @@ public PetsController(IPetService petService)
         }
 
         [Produces(typeof(ServiceResponse<PetsGetPetsDtoResponse>))]
+        [AuthorizeRoles(Roles = Role.Administrator)]
         [HttpGet]
+        [AuthorizeRoles(Roles = Role.Administrator)]
         public async Task<IActionResult> GetPets()
         {
             var response = await _petService.GetPetsAsync();
             return SendResponse(response);
         }
 
+        [Produces(typeof(ServiceResponse<PetsGetMyPetsDtoResponse>))]
+        [AuthorizeRoles(Roles = Role.Owner)]
```

Comment:
Czy w przypadku pojedynczej roli też robimy to za pomocą customoweg filtru? Tzn nigdzie tego w sumie nie określiliśmy i widziałem już obie wersje. Warto uściślić

Reply:
rzeczywiscie, dam tutaj Authorize od aspneta